http/src/main/java/com/redhat/lightblue/client/http/LightblueHttpClient.java

```diff
@@ -405,4 +407,17 @@ private static HttpTransport defaultHttpClientFromConfig(LightblueClientConfigur
         }
     }
 
+    @Override
+    public LightblueHealthResponse lightblueHealth(LightblueHealthRequest lightblueHealthRequest)
```

Comment:
lightblue is probably redundant here. Maybe just "diagnostics()"?

There's probably not a need to pass a request object for diagnostics or health either because they don't have any parameters.